### ISSUE TITLE
Reimplement link interface port

### ISSUE THREAD

**Issue by andrelbd1** [TeleMidia/ginga#55]
Application example: **comerciais-proview**

```
<context id="ctxUnidades">
        <port id="ptoBtnUnidadesVolta" component="imgBtnUnidadesVolta" />
        <media descriptor="dsHeadUnidades" id="imgHeadUnidades" src="media/imagens/condominio/unidades/unidadesHead.png"/>
</context>

<link xconnector="onKeySelectionStopNStartN">
        <bind role="onSelection" component="ctxUnidades" interface="ptoBtnUnidadesVolta" >
          <bindParam name="keyCode" value="GREEN"/>
        </bind>
        <bind component="ctxUnidades" role="stop"/>
        <bind component="ctxCondicoes" role="start"/>
</link>
```

**Comment by andrelbd1** [comment]
@gflima

**Comment by alanlivio** [comment]
Duplicated and resolved by #95
